samples/renode_example/renode/STTS751/README.md

```diff
@@ -0,0 +1,4 @@
+# STTS751
+
+This document outlines the steps required to implement a custom peripheral for
+Phenode. This chip is supported by Zephyr, but is not implemented in Renode.
```

Comment:
Add the following:
```
1. Add I2C device to .overlay file - driver must be supported in Zephyr
2. Add sensor to .repl file, using the same address as in .overlay file
3. Add Sensor implementation to STTS751.cs. There are several ways to implement the
sensor. We have followed the example of LIS2DW12.cs. Several methods must be implemented, this
depends on the class the sensor inherits from.
4. Include the STTS751.cs sensor in the .resc file using the `include` syntax
5. Add a simple test to the .robot file

Some useful information regarding the implementation can be found here:
* [Renode - Writing a new peripheral](https://renode.readthedocs.io/en/latest/advanced/writing-peripherals.html#)
* [LIS2DW12 .cs implementation](https://github.com/renode/renode-infrastructure/blob/master/src/Emulator/Peripherals/Peripherals/Sensors/LIS2DW12.cs)
* [Step by Step Implementation guide by betrusted[(https://betrusted.io/xous-book/ch04-04-writing-cs-peripherals.html)
```

Reply:
I moved this text from the top of the STTS751.cs file to this Readme.